packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space/allocate_disk_space_dialogs.dart

```diff
@@ -0,0 +1,326 @@
+                  ),
+                ),
+                Expanded(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.stretch,
+                    children: <Widget>[
+                      ConstrainedBox(
+                        constraints: BoxConstraints(maxHeight: tileHeight),
+                        child: _PartitionSizeBox(
+                          bytes: partitionSize,
+                          unit: partitionUnit,
+                          freeSpace: disk.freeSpace,
+                        ),
+                      ),
+                      const SizedBox(height: kContentSpacing),
+                      _RadioValueTile(
+                        title: Text(lang.partitionTypePrimary),
+                        value: PartitionType.primary,
+                        groupValue: partitionType,
+                      ),
+                      _RadioValueTile(
+                        title: Text(lang.partitionTypeLogical),
+                        value: PartitionType.logical,
+                        groupValue: partitionType,
+                      ),
+                      const SizedBox(height: kContentSpacing),
+                      _RadioValueTile<PartitionLocation>(
+                        title: Text(lang.partitionLocationBeginning),
+                        value: PartitionLocation.beginning,
+                        groupValue: partitionLocation,
+                      ),
+                      _RadioValueTile<PartitionLocation>(
+                        title: Text(lang.partitionLocationEnd),
+                        value: PartitionLocation.end,
+                        groupValue: partitionLocation,
+                      ),
+                      const SizedBox(height: kContentSpacing),
+                      ConstrainedBox(
+                        constraints: BoxConstraints(maxHeight: tileHeight),
+                        child: ValueListenableBuilder<PartitionFormat>(
+                          valueListenable: partitionFormat,
+                          builder: (context, type, child) {
+                            return DropdownBuilder<PartitionFormat>(
+                              selected: type,
+                              values: PartitionFormat.values,
+                              itemBuilder: (context, type, _) {
+                                return Text(type.localize(context));
+                              },
+                              onSelected: (value) {},
+                            );
+                          },
+                        ),
+                      ),
+                      const SizedBox(height: kContentSpacing),
+                      ConstrainedBox(
+                        constraints: BoxConstraints(maxHeight: tileHeight),
+                        child: Autocomplete<String>(
+                          initialValue: TextEditingValue(
+                            text: lang.partitionMountPointAutomatic,
```

Comment:
This makes use of https://github.com/flutter/flutter/pull/80257, which isn't available on flutter stable yet.